09_employment/employment_metrics_place.qmd

```diff
@@ -0,0 +1,575 @@
+```{r}
+metrics_employment <- metrics_employment %>%
+  arrange(year, statefip, place)
+```
+
+Merge the data quality flags and create the final data quality metric based on both size and geographic allocation.
+```{r}
+metrics_employment <- left_join(metrics_employment, place_puma, by=c("crosswalk_period", "statefip","place"))
+```
+
+Generate the quality variable for each place.
+```{r}
+metrics_employment <- metrics_employment %>% 
+  mutate(share_employed_quality = case_when(size_flag==0 & geographic_allocation_quality==1 ~ 1,
+                                     size_flag==0 & geographic_allocation_quality==2 ~ 2,
+                                     size_flag==0 & geographic_allocation_quality==3 ~ 3,
+                                     size_flag==1 ~ 3)
+  )
+```
+
+Keep only relevant variables.
+```{r}
+
+metrics_employment <- metrics_employment %>%
+  select(year,
+         statefip,
+         place,
+         subgroup_type, 
+         subgroup,
+         share_employed,
+         share_employed_quality)
+
+```
+
+### Evaluate results
+
+Look at distribution of employment share by place.
+
+```{r}
+
+metrics_employment %>% 
+  filter(year == 2022, subgroup == "All") %>% 
+  select(share_employed) %>% 
+  ggplot(aes(x = share_employed)) +
+  geom_density(alpha = 0.15) +
+  theme_minimal() +
+  ggtitle("Distribution of share employed, by place 2022") +
+  ylab("Density")
+  
+```
+
+Look at distribution of employment share by place by race.
+```{r}
+
+metrics_employment %>% 
+  filter(year == 2021, subgroup == "Black, Non-Hispanic") %>% 
+  select(place, share_employed) %>% 
+  ggplot(aes(x = share_employed)) +
+  geom_density(alpha = 0.15) +
+  theme_minimal() +
+  ggtitle("Employment Share, Black") +
+  ylab("Density")
+
+metrics_employment %>% 
+  filter(year == 2021, subgroup == "Hispanic") %>% 
+  select(place, share_employed) %>% 
+  ggplot(aes(x = share_employed)) +
+  geom_density(alpha = 0.15) +
+  theme_minimal() +
+  ggtitle("Employment Share, Hispanic") +
+  ylab("Density")
+
+metrics_employment %>% 
+  filter(year == 2021, subgroup == "Other Races and Ethnicities") %>% 
+  select(place, share_employed) %>% 
+  ggplot(aes(x = share_employed)) +
+  geom_density(alpha = 0.15) +
+  theme_minimal() +
+  ggtitle("Employment Share, Other Races and Ethnicities") +
+  ylab("Density")
+
+metrics_employment %>% 
+  filter(year == 2021, subgroup == "White, Non-Hispanic") %>% 
+  select(place, share_employed) %>% 
+  ggplot(aes(x = share_employed)) +
+  geom_density(alpha = 0.15) +
+  theme_minimal() +
+  ggtitle("Employment Share, White, Non-Hispanic") +
+  ylab("Density")
+
+```
+
+Look at the counts of the quality flag.
+```{r}
+metrics_employment %>% 
+  select(share_employed_quality) %>% 
+  ggplot(aes(x = share_employed_quality)) +
+  geom_histogram(color = "blue", fill = alpha("blue", 0.3), binwidth = 1.0) +
+  theme_minimal() +
+  ggtitle("Quality Flag") +
+  ylab("Count")
```

Comment:
Can you look at this for `"All"` and subgroups separately? Repeat for all metrics.